src/main/scala/nl.knaw.dans.easy.curationwork/Command.scala

```diff
@@ -28,22 +27,35 @@ import scala.util.{ Failure, Success, Try }
 object Command extends App with DebugEnhancedLogging {
   type FeedBackMessage = String
 
-  val configuration = Configuration(Paths.get(System.getProperty("app.home")))
+  val configuration = Configuration(Paths.get(System.getProperty("app.home")), "application.properties")
+  val cfgDatamanagers = Configuration(Paths.get(System.getProperty("app.home")), "datamanager.properties")
   val commandLine: CommandLineOptions = new CommandLineOptions(args, configuration) {
     verify()
   }
-  val app = new EasyManageCurationWorkApp(configuration)
+  val app = new EasyManageCurationWorkApp(configuration, cfgDatamanagers)
 
   runSubcommand(app)
-    .doIfSuccess(msg => println(s"OK: $msg"))
+    .doIfSuccess(msg => println(s"$msg"))
     .doIfFailure { case e => logger.error(e.getMessage, e) }
     .doIfFailure { case NonFatal(e) => println(s"FAILED: ${ e.getMessage }") }
 
   private def runSubcommand(app: EasyManageCurationWorkApp): Try[FeedBackMessage] = {
     commandLine.subcommand
       .collect {
-        case cmd @ commandLine.list => Success("Message") // handle list
+        case cmd @ commandLine.list =>
+          if (validDatamanager(cmd.datamanager.toOption)) app.listCurationWork(cmd.datamanager.toOption)
+          else Try(s"Error: Unknown datamanager ${cmd.datamanager.apply()}")
+        case cmd @ commandLine.assign =>
+          if (validDatamanager(cmd.datamanager.toOption)) app.assignCurationWork(cmd.datamanager.toOption, cmd.uuid.toOption)
+          else Try(s"Error: Unknown datamanager ${cmd.datamanager.apply()}")
+        case cmd @ commandLine.unassign =>
+          if (validDatamanager(cmd.datamanager.toOption)) app.unassignCurationWork(cmd.datamanager.toOption, cmd.uuid.toOption)
+          else Try(s"Error: Unknown datamanager ${cmd.datamanager.apply()}")
       }
       .getOrElse(Failure(new IllegalArgumentException(s"Unknown command: ${ commandLine.subcommand }")))
   }
+
+  private def validDatamanager(datamanager: Option[DatamanagerId]): Boolean = {
+    datamanager.isEmpty || cfgDatamanagers.properties.containsKey(datamanager.getOrElse(""))
+  }
```

Comment:
Yes, in unassign you can leave the datamanager out (and then  also the uuid).  if datamanager is left out, datamanager is the current user.

Reply:
Additional question (because you're running `whoami` in that case): do you make the assumption that the username of the server account is equal to the account used in the configuration? I'd say the latter one is the one defined in LDAP (not sure)? Maybe @janvanmansum can say something about this?

Reply:
Datamanager ids used in this application are unix-user ids